src/mlpack/methods/naive_bayes/naive_bayes_classifier_impl.hpp

```diff
@@ -384,8 +384,7 @@ void NaiveBayesClassifier<ModelMatType>::Classify(
   // Now calculate maximum probabilities for each point.
   for (size_t i = 0; i < data.n_cols; ++i)
   {
-    arma::uword maxIndex = 0;
-    logLikelihoods.unsafe_col(i).max(maxIndex);
+    arma::uword maxIndex = logLikelihoods.unsafe_col(i).index_max();
```

Comment:
this file also needs similar change on line 262

Reply:
Good catch.  I could have simply applied that change, no need for all the machinery of a full fork and PR for a one-line change. 

The more important part is that this appears to have fixed the prediction test fail as the indices are now used consistently.  So yay.

Reply:
@eddelbuettel The requested change is specifically the `naive_bayes_classifier_impl.hpp` file.
The PR was for a different file: `perceptron_impl.hpp`  ([related commit](https://github.com/conradsnicta/fix-perceptron_impl/commit/9ed1f4b04d14cc69d6f6f7a516e39482d5fea432)).

Reply:
There are likely a million more.  What I set out to do was to addresss the nag listed in the initial comment (https://github.com/mlpack/mlpack/pull/3835#issue-2672510534) and to "not do harm".  I never claimed to catch all instances.

I can of course easily address another one-off change but there may be more efficient ways.